src/AcmCharts/AcmDonutChart/AcmDonutChart.stories.tsx

```diff
@@ -47,6 +53,16 @@ export const DonutChart = () => {
             />
             <AcmDonutChart title="Pods" description="Overview of pod count and status" data={podData} />
             <AcmDonutChart title="Cluster status" description="Overview of cluster status" data={clusterData} />
+            <AcmDonutChart
+                title="Cluster issues"
+                description="Overview of cluster issues"
+                data={insightData}
+                donutTitle={{
```

Comment:
Minor suggestion: How about using `donutLabel` instead to make it clear that it's different from the card title?